en/docs/get-started/quick-start-guide.md

```diff
@@ -104,19 +103,50 @@ Follow this procedure to further develop the API by adding a new resource and si
 
      ![Configure Variable](../assets/img/get-started/configure-variable.png){.cInlineImage-full}
 
-8. Click **Save**.
+ 7. Click **Save**.
 
-9. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
-   ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
-   ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
+ 8. To return the personalized message, click **+** below the variable you created, click **Return**, and enter `message` as the **Return Expression**.
+    ![Add Return Type](../assets/img/get-started/add-return-type.png){.cInlineImage-full}
+    ![Select Return](../assets/img/get-started/select-return.png){.cInlineImage-full}
    
-10. Click **Save**.
+ 9. Click **Save**.
 
 !!!info
-    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can choose to work in the low-code view or pro-code view, or both simultaneously.    
+    You can click the **</>** icon on the top left corner of the editor if you want to work in the code view. Depending on your expertise, you can work in the low-code view or the pro-code view. You can also work in both views simultaneously.    
     ![Editor View](../assets/img/get-started/low-code-pro-code-side-by-side.png){.cInlineImage-full}
```

Comment:
Looks like the text on the left side has got selected by mistake on taking the screenshot. Shall we retake and add the screenshot?

Reply:
Done